docs/conf.py

```diff
@@ -30,7 +30,8 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ["jupyter_sphinx.execute", "sphinx_rtd_theme"]
+
```

Comment:
Why an extra new line?

Reply:
I did the merge in the gihub interface, so I didn't checked. Let me remove it